testsuite/pom.xml

```diff
@@ -118,6 +119,11 @@
                         <groupId>javax.inject</groupId>
                         <artifactId>javax.inject</artifactId>
                     </exclusion>
+                    <!-- caused  [CIRCULAR REFERENCE: java.lang.RuntimeException: Could not create new instance of class org.jboss.arquillian.test.impl.EventTestRunnerAdaptor] -->
```

Comment:
I'll file a bug here for WFARQ. There is no reason for this dependency. Thank you for finding it!